docs/reference/data-streams/downsampling-manual.asciidoc

```diff
@@ -0,0 +1,450 @@
+[[downsampling-manual]]
+=== Run downsampling manually
+++++
+<titleabbrev>Run downsampling manually</titleabbrev>
+++++
+
+preview::[]
+
+This is a simplified example that allows you to see quickly how <<downsampling,downsampling>> works to reduce the storage size of a sampled set of metrics. The example uses typical Kubernetes system monitoring data. To test out downsampling, follow these steps:
+
+. Check the <<downsampling-manual-prereqs,prerequisites>>.
+. <<downsampling-manual-create-index>>.
+. <<downsampling-manual-ingest-data>>.
+. <<downsampling-manual-run>>.
+. <<downsampling-manual-view-results>>.
+
+[discrete]
+[[downsampling-manual-prereqs]]
+==== Prerequisites
+
+Refer to <<tsds-prereqs,time series data stream prerequisites>>.
+
+For the example you need a sample data file. Download the file from link:
+https://static-www.elastic.co/v3/assets/bltefdd0b53724fa2ce/bltf2fe7a300c3c59f7/631b4bc5cc56115de2f58e8c/sample-k8s-metrics.json[here] and save it in the local directory where you're running {es}.
+
+[discrete]
+[[downsampling-manual-create-index]]
+==== Create a time series index
+
+This creates an index template for a basic data stream. The available parameters for an index template are described in detail in <<set-up-a-data-stream,Set up a time series data stream>>.
```

Comment:
This creates the actual index, not an index template

Reply:
Changed to `This creates an index for a basic data stream. The available parameters for an index are described in detail in <<set-up-a-data-stream,Set up a time series data stream>>.` in [507419d](https://github.com/elastic/elasticsearch/pull/88571/commits/507419dcc6e4498ad50f37505d150e5de4db8a77)